contentwarehouse/snippets/set_acl_sample.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_set_acl]
+
+from typing import Dict
+
+from google.cloud import contentwarehouse
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_number = 'YOUR_PROJECT_NUMBER'
+# location = 'YOUR_PROJECT_LOCATION' # Format is 'us' or 'eu'
+# document_id = 'YOUR_DOCUMENT_ID'
+# user_id = 'user:YOUR_SERVICE_ACCOUNT_ID' # Format is "user:xxxx@example.com"
+# policy = "Policy in JSON format"
+
+
+def set_acl(
+    project_number: str,
+    location: str,
+    policy: Dict[str,list[Dict]],
```

Comment:
What is the inner `Dict`? What is the nested structure? The users should be able to know what the structure is from the type hint, rather than a generic `Dict`.

If you're unsure about the type that is expected because it is dynamic, consider using `Any`:

```
policy: Dict[str, List[Dict[str, Any]]]
```
Which will denote that there is a list of dictionary that contains a `str` mapping to an arbitrary / dynamic structure inside.

Reply:
Thanks, this makes sense.